libfreerdp/core/server.h

```diff
@@ -66,6 +66,8 @@ struct rdp_peer_channel
 	BYTE dvc_open_state;
 	UINT32 dvc_total_length;
 	rdpMcsChannel* mcsChannel;
+
+	char name[128];
```

Comment:
Perhaps it should be `channelName` to match other fields and we can probably move it with other `channelX` fields as I'm not sure we have much to care about ABI here.

Reply:
good idea